hq/app/controllers/HQController.scala

```diff
@@ -8,6 +8,8 @@ import play.api.libs.ws.WSClient
 import play.api.mvc._
 import services.CacheService
 import utils.attempt.PlayIntegration.attempt
+import logic.ReportDisplay.sortAccountsByReportSummary
```

Comment:
Mentioned below, but sortAccountsByReportSummary should probably not be in ReportDisplay...

Reply:
+1